docs/guides/high-availability/concepts-multi-site.adoc

```diff
@@ -60,42 +56,41 @@ Monitoring is necessary to detect degraded setups.
 | Less than one minute
 
 | {jdgserver_name} cluster failure
-| If the {jdgserver_name} cluster fails in the active site, {project_name} will not be able to communicate with the external {jdgserver_name}, and the {project_name} service will be unavailable.
-The loadbalancer will detect the situation as `/lb-check` returns an error, and will fail over to the other site.
+| If the {jdgserver_name} cluster fails in one of the sites, {project_name} will not be able to communicate with the external {jdgserver_name} on that site, and the {project_name} service will be unavailable.
+The loadbalancer will detect the situation as `/lb-check` returns an error, and will direct all traffic to the other site.
 
-The setup is degraded until the {jdgserver_name} cluster is restored and the session data is re-synchronized to the primary.
+// TODO currently the failed site will automatically resume with out-of-sync state
```

Comment:
With the lambda we have in place, AFAIK the site will be taken offline in LB if that happens?

Reply:
The A/A Lambda only handles split-brain scenarios atm. The current A/P Lambda in the docs updates the Route53 records. If we want a similar behaviour for the new multi-site, which I think we do, we need to either add a second Lambda or update the stonith one to handle this scenario.

Reply:
Oh, I see. Hm. Let's take this as a separate issue then. Using the same lambda might be a thing if we can generate a similar alert from Prometheus for that.

Reply:
Actually, thinking about it more, I'm not sure if it is an issue or not.

I think it comes down to how long Infinispan/JGroups takes to detect a split-brain compared with how long it takes the accelerator to mark a site as offline. The current health check settings define on our NLBs is a healthcheck-interval of 10s and a healthcheck-unhealthy-threshold of 3, resulting in at least a 30s delay before unhealthy Keycloak pods are detected and traffic re-routed.

@pruivo Do you know ~ how long it should take for JGroups to detect the split-brain with the current config?